kateryna/daemon/parsing_logs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+
+ advanced_syslog() {
+   function log {
+        while read LINE; do
+                  if [[ $LINE == "6"* ]]; then
```

Comment:
Different log levels `https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels`